docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -191,6 +201,18 @@ The `allowed_domains` option limits access to users who belong to specific domai
 allowed_domains = mycompany.com mycompany.org
 ```
 
+### PKCE
+
+> Available in Grafana v8.3 and later versions.
+
+IETF's [RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636)
+introduces "proof key for code exchange" (PKCE) which introduces
+additional protection against some forms of authorization code
+interception attacks. PKCE will be required in [OAuth 2.1](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-03).
+
+> **Note:** Grafana v10.1 and newer use `use_pkce = true` by default,
+> You can disable PKCE in Grafana by setting `use_pkce` to `false` in the`[auth.azuread]` section.
+
 ### Configure automatic login
 
 Set `auto_login` option to true to attempt login automatically, skipping the login screen.
```

Comment:
```suggestion
If you want to automatically log in without having to go through the login screen, enable the `auto_login` option.
```

Reply:
This was not included in my changes.
Are you sure you want to make this change? (If you👍 then I will do.)

Reply:
@arukiidou , my apologies, I didn't see this one. I adjusted the suggestion slightly. Feel free to commit it.